syntax = "proto3";

package kornia.protos;

message ImageSize {
    int32 height = 1;
    int32 width = 2;
}

message Image {
    bytes image_data = 1;
    int64 frame_number = 2;
    string encoding_type = 3;
    ImageSize image_size = 4;
}